app/frontend/components/domains/jurisdictions/grid-header.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Flex, GridItem, Text, styled } from "@chakra-ui/react"
+import { observer } from "mobx-react-lite"
+import React from "react"
+import { useTranslation } from "react-i18next"
+import { useMst } from "../../../setup/root"
+import { EJurisdictionSortFields } from "../../../types/enums"
+import { SearchInput } from "../../shared/base/search-input"
+import { SortIcon } from "../../shared/sort-icon"
+
+export const GridHeaders = observer(function GridHeaders() {
+  const { jurisdictionStore } = useMst()
+  const { sort, getSortColumnHeader, toggleSort } = jurisdictionStore
+  const { t } = useTranslation()
+
+  return (
+    <Box display={"contents"} role={"rowgroup"}>
+      <Box display={"contents"} role={"row"}>
+        <GridItem
+          as={Flex}
+          gridColumn={"span 6"}
+          p={6}
+          bg={"greys.grey10"}
+          justifyContent={"space-between"}
+          align="center"
+        >
+          <Text role={"heading"} as={"h3"} color={"black"} fontSize={"sm"} height="fit-content">
+            {t("jurisdiction.index.tableHeading")}
+          </Text>
+          <SearchInput searchModel={jurisdictionStore} />
```

Comment:
Maybe the search input needs to be absolutely positioned? It seems like the columns in jurisdiction are too wide so the search input gets hidden by horizontal scroll. I might need to make the same change in requirements library page, but the columns there don't get as large so it seems to work fine.


https://github.com/bcgov/HOUS-permit-portal/assets/32022335/d14b7774-2d58-446d-ae95-1b9d34e76d73

Reply:
It looks like the translations got picked out. Will fix